core/scripts/bareos-config-lib.sh.in

```diff
@@ -234,6 +234,78 @@ get_database_utility_path()
     fi
 }
 
+_scsicrypto_cmds="bareos-sd bcopy bextract bls bscan bscrypto btape"
+_scsicrypto_config_file="${BAREOS_CONFIG_DIR}/.enable-capabilities"
+check_scsicrypto_capabilities()
+{
+  if [ "${os_type}" != "Linux" ];then
+      error "check_scsicrypto_capabilities() is not supported on this platform"
+      return 1
+  fi
+  if ( [ "$(command -v getcap)" = "" ] || [ "$(command -v setcap)" = "" ] ); then
+      error "getcap and setcap commands are mandatory"
+      return 1
+  fi
+  if [ -f "${_scsicrypto_config_file}" ];then
+    cap_missing_conf=0
+  else
+    cap_missing_conf=1
+  fi
+  for c in ${_scsicrypto_cmds};do
+    if ! r="$(setcap -v cap_sys_rawio=ep "${BAREOS_SBIN_DIR}/${c}")"; then
+       warn "${r}"
+       cap_missing_conf=1
+    fi
+  done
+  if [ ${cap_missing_conf} -ne 0 ];then
+    warn "One of the tools are not configured for scsi crypto."
+    warn "Run set_scsicrypto_capabilities to fix capabilities."
+  else
+    info "All tools have cap_sys_rawio=ep set."
+  fi
+}
+
+set_scsicrypto_capabilities()
+{
+  if [ "${os_type}" != "Linux" ];then
+      error "set_scsicrypto_capabilities() is not supported on this platform"
+      return 1
+  fi
+  for c in ${_scsicrypto_cmds};do
+    if ! r="$(setcap cap_sys_rawio=ep "${BAREOS_SBIN_DIR}/${c}")"; then
+       error "setcap on ${c} has failed - ${r}"
+       return 1
+    else
+      if ! r="$(chmod 0750 "${BAREOS_SBIN_DIR}/${c}")"; then
+         error "Adjusting chmod failed on ${c}"
+         return 1
+      fi
+    fi
+  done
+  echo "cap_sys_rawio+ep capabilities enabled" > "${_scsicrypto_config_file}"
+}
+
+unset_scsicrypto_capabilities()
+{
+  if [ "${os_type}" != "Linux" ];then
+      error "unset_scsicrypto_capabilities() is not supported on this platform"
+      return 1
+  fi
+  for c in ${_scsicrypto_cmds};do
+    if ! r="$(setcap -r "${BAREOS_SBIN_DIR}/${c}")"; then
+       error "setcap -r on ${c} has failed - ${r}"
+       return 1
+    else
+      if ! r="$(chmod 0755 "${BAREOS_SBIN_DIR}/${c}")"; then
```

Comment:
I'd not try to change back the permission. Maybe there have been other requirements in between.

Reply:
Ok I'm not a fanboy of a unset not doing exactly the reverse of what set do, it's a door open to support request.
I'm more expecting to have an administrator able to remember what other requirements he set ...

Reply:
Ideally we would have use packaging tools, for example ```rpm --restore <packagename>``` which set back ownership, mode and capabilities, but there's absolutely no equivalent in .deb world the package database doesn't allow this. 
```apt reinstall <packagename> ``` can emulate in a way but it really change all files present in the package (which has others implications)

Reply:
In DEB packages you don't have permission handling as in RPM. Currently we use the post install script to set permissions (on configuration files).